ydb/tests/functional/canonical/canondata/test_sql.TestCanonicalFolder1.test_case_write_multi_usage.script-script_/write_multi_usage.script.plan

```diff
@@ -214,98 +214,57 @@
                 "PlanNodeType": "Query",
                 "Plans": [
                     {
-                        "Node Type": "Effect",
-                        "PlanNodeId": 12,
+                        "Node Type": "Sink",
+                        "PlanNodeId": 2,
                         "Plans": [
                             {
-                                "CTE Name": "precompute_0_1",
-                                "Node Type": "Upsert-ConstantExpr",
+                                "Node Type": "ConstantExpr-Sink",
                                 "Operators": [
-                                    {
-                                        "Inputs": [
-                                            {
-                                                "InternalOperatorId": 1
-                                            }
-                                        ],
-                                        "Name": "Upsert",
-                                        "Path": "/local/base_write_multi_usage_script_script/Temp",
-                                        "Table": "base_write_multi_usage_script_script/Temp"
-                                    },
                                     {
                                         "Inputs": [],
-                                        "Iterator": "precompute_0_1",
+                                        "Iterator": "[{Group: 1,Name: ToString}]",
```

Comment:
Don't print data to plan

Reply:
Looks like it's expected to be printed this way...

https://github.com/ydb-platform/ydb/blob/cb28e2ee8bd69b570b6caf81256815382a616a7b/ydb/tests/functional/canonical/canondata/test_sql.TestCanonicalFolder1.test_case_simple_script_params.script-script_/simple_script_params.script.plan#L22

https://github.com/ydb-platform/ydb/blob/cb28e2ee8bd69b570b6caf81256815382a616a7b/ydb/tests/functional/canonical/canondata/test_sql.TestCanonicalFolder1.test_case_explain.script-script_/explain.script.plan#L33